test/system/test_misc.py

```diff
@@ -546,3 +546,56 @@ def test_error_raised_when_no_IC_heat_transfer():
         match="Initial condition is required for transient heat transfer simulations",
     ):
         my_model.initialise()
+
+
+def test_catch_bug_738(tmpdir):
+    """
+    Test to catch bug #738
+    Set up a simple simulation with and XDMFExport in write mode "last"
```

Comment:
Maybe, just `with` or `and`?

Reply:
Good catch, this was a typo and should've been "with an XDMFExport"

Reply:
Will you merge this PR?

Reply:
Yep unless you have other comments?

Reply:
Nope